### PR TITLE
Added first scenario

### PR DESCRIPTION
**Pull request by AnnaLangner**

### CODE REVIEW

File: script/test_todo.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import NoSuchAttributeException
+
+
+def main():
```
Comment (klangner) [inline script/test_todo.py:48]: Let's start fixing this code by some rafactoring:
Try to refactor function main into a list of functions which would resemble the list of steps from the issue.

For example:
  * Add "task1" and press enter

The code for this should looks like this:
```python
def add_task(driver, title):
   input_todo = driver.find_element_by_xpath("//html/body/section/div/header/input")
   input_todo.clear()
   input_todo.send_keys("task1")
   input_todo.send_keys(Keys.RETURN)


def main():
   driver = ...
   add_task(driver, "task1")
  # Rest of the actions
```

Create actions for all steps and use them in function main.

**Review by klangner — CHANGES_REQUESTED** [review]

File: script/test_todo.py (hunk 1 above)
Comment (AnnaLangner) [reply script/test_todo.py:48]: Ok :) I was just wondering how to solve the problem of repetitive code. Now I know .

File: script/test_todo.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  assert elem.get_attribute("class") == "completed"
+
+
+
+def main():
+  driver = webdriver.Chrome(executable_path="bin/chromedriver")
+  driver.get("http://todomvc.com/examples/react/#/")
+  WebDriverWait(driver, 10).until(EC.presence_of_element_located((By.XPATH, "//html/body/section/div/header/input")))
+  add_task(driver, "task1")
+  active_task(driver, 1)
```
Comment (klangner) [inline script/test_todo.py]: The second step is: "Check if "task1" is active"
So the name of the function should relate to this. Better name is:
"verify_task_active" or something similar.

This function should also use `assert` keyword to check this assert

File: script/test_todo.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+def add_task(driver, title):
+  input_todo = driver.find_element_by_xpath("//html/body/section/div/header/input")
+  input_todo.clear()
+  input_todo.send_keys(title)
+  input_todo.send_keys(Keys.RETURN)
+
+
+def active_task(driver, id):
+  elem = driver.find_element_by_xpath("/html/body/section/div/section/ul/li[%s]"%str(id))
+  try:
```
Comment (klangner) [inline script/test_todo.py]: Refactor this code to use `assert`

File: script/test_todo.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+def add_task(driver, title):
+  input_todo = driver.find_element_by_xpath("//html/body/section/div/header/input")
+  input_todo.clear()
+  input_todo.send_keys(title)
+  input_todo.send_keys(Keys.RETURN)
+
+
+def active_task(driver, id):
+  elem = driver.find_element_by_xpath("/html/body/section/div/section/ul/li[%s]"%str(id))
```
Comment (klangner) [inline script/test_todo.py]: Use string interpolation here.
Example:
```python
f'/html/body/section/div/section/ul/li[{id}]'
```

File: script/test_todo.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  try:
+    class_elem = elem.get_attribute("class")
+    if(class_elem == "completed"):
+      return True
+  except:
+    NoSuchAttributeException
+
+
+def comparison_of_adjacent_tasks(driver, id):
```
Comment (klangner) [inline script/test_todo.py]: Rename this function to 'verify_...'
Use assert inside.

File: script/test_todo.py [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  elem_1 = driver.find_element_by_xpath("/html/body/section/div/section/ul/li[%s]"%str(id))
+  elem_2 = driver.find_element_by_xpath("/html/body/section/div/section/ul/li[%s]"%str(id+1))
+  if(elem_1.text != elem_2.text):
+    pass
+  else:
+    print("You already have this task")
+
+
+def task_completed(driver, id):
```
Comment (klangner) [inline script/test_todo.py]: Do not combine 2 functionalities in one function.
It shouldn't click and verify. Split into 2 different functions

**Review by klangner — CHANGES_REQUESTED** [review]

File: script/test_todo.py [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  assert class_elem != "completed"
+
+
+def verify_comparison_of_adjacent_tasks(driver, id):
+  elem_1 = driver.find_element_by_xpath(f"/html/body/section/div/section/ul/li[{id}]")
+  elem_2 = driver.find_element_by_xpath(f"/html/body/section/div/section/ul/li[{id}+{1}]")
+  assert elem_1.text != elem_2.text  
+
+
+def click_on_input_task_completed(driver, id):
```
Comment (klangner) [inline script/test_todo.py]: Better name: `click_task`

File: script/test_todo.py [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+  input_todo.send_keys(Keys.RETURN)
+
+
+def verify_active_task(driver, id):
+  elem = driver.find_element_by_xpath(f"/html/body/section/div/section/ul/li[{id}]")
+  class_elem = elem.get_attribute("class")
+  assert class_elem != "completed"
+
+
+def verify_comparison_of_adjacent_tasks(driver, id):
```
Comment (klangner) [inline script/test_todo.py]: Better function:
`verify_tasks_different(driver, id1, id2)

**Review by klangner — CHANGES_REQUESTED** [review]

File: script/test_todo.py [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+  elem_toggle.click()
+
+
+def verify_task_completed(driver, id):
+  elem = driver.find_element_by_xpath(f"/html/body/section/div/section/ul/li[{id}]")
+  assert elem.get_attribute("class") == "completed"
+
+
+
+def main():
```
Comment (klangner) [inline script/test_todo.py]: It looks ok. Just last one change which will help us in the future.

Rename function `main` into `test_task_completed` and then create new `main` function witch will only call this `test_task_completed` function. So every scenario will have it own function and those function will be called from main one by one.

**Review by klangner — CHANGES_REQUESTED** [review]

**Review by klangner — APPROVED** [review]

**Pull request merged**